src/Toolkit/I18n.php

```diff
@@ -190,6 +198,20 @@ public static function translations(): array
         return static::$translations;
     }
 
+    /**
+     * Returns (and creates) a decimal number formatter for a given locale
+     *
+     * @return array
+     */
+    protected static function decimalNumberFormatter(string $locale): NumberFormatter
```

Comment:
```suggestion
    protected static function decimalNumberFormatter(string $locale): ?NumberFormatter
```

Reply:
Should we not remove the type hint in general as we didn't want to hint for objects/classes?

Reply:
In this case it's fine as it's a protected static method inside a class that doesn't get extended. Object type hints are only a problem in inheritance.

I thought about it as well, but I'd leave the hinting in as we'll be able to use object type hints everywhere in PHP 8. So starting with it in places like this where it already works well will reduce the migration effort later.